src/xkit.cs

```diff
@@ -0,0 +1,1144 @@
+#if XAMCORE_4_0 || MONOMAC
+		[Sealed]
+#endif
+		[Export ("textContainerForGlyphAtIndex:effectiveRange:withoutAdditionalLayout:")]
+		NSTextContainer GetTextContainer (nuint glyphIndex, /* nullable NSRangePointer */ out NSRange effectiveGlyphRange, bool withoutAdditionalLayout);
+
+		[Export ("usedRectForTextContainer:")]
+#if XAMCORE_4_0
+		CGRect GetUsedRect (NSTextContainer container);
+#else
+		CGRect GetUsedRectForTextContainer (NSTextContainer container);
+#endif
+
+		/* GetLineFragmentRect (NSUInteger, NSRangePointer) */
+		[Protected]
+		[Export ("lineFragmentRectForGlyphAtIndex:effectiveRange:")]
+		CGRect GetLineFragmentRect (nuint glyphIndex, /* nullable NSRangePointer */ IntPtr effectiveGlyphRange);
+
+		[Wrap ("GetLineFragmentRect (glyphIndex, IntPtr.Zero)")]
+		CGRect GetLineFragmentRect (nuint glyphIndex);
+
+		[Sealed]
+		[Export ("lineFragmentRectForGlyphAtIndex:effectiveRange:")]
+		CGRect GetLineFragmentRect (nuint glyphIndex, out /* nullable NSRangePointer */ NSRange effectiveGlyphRange);
+
+		/* GetLineFragmentRect (NSUInteger, NSRangePointer, bool) */
+		[iOS (9,0)]
+#if MONOMAC || XAMCORE_4_0
+		[Protected]
+#else
+		[Sealed][Internal]
+#endif
+		[Export ("lineFragmentRectForGlyphAtIndex:effectiveRange:withoutAdditionalLayout:")]
+		CGRect GetLineFragmentRect (nuint glyphIndex, /* nullable NSRangePointer */ IntPtr effectiveGlyphRange, bool withoutAdditionalLayout);
+
+		[iOS (9,0)]
+		[Wrap ("GetLineFragmentRect (glyphIndex, IntPtr.Zero)")]
+		CGRect GetLineFragmentRect (nuint glyphIndex, bool withoutAdditionalLayout);
+
+		[iOS (9,0)]
+#if MONOMAC || XAMCORE_4_0
+		[Sealed]
+#endif
+		[Export ("lineFragmentRectForGlyphAtIndex:effectiveRange:withoutAdditionalLayout:")]
+		CGRect GetLineFragmentRect (nuint glyphIndex, out /* nullable NSRangePointer */ NSRange effectiveGlyphRange, bool withoutAdditionalLayout);
+
+		/* GetLineFragmentUsedRect (NSUInteger, NSRangePointer) */
+		[Protected]
+		[Export ("lineFragmentUsedRectForGlyphAtIndex:effectiveRange:")]
+		CGRect GetLineFragmentUsedRect (nuint glyphIndex, /* nullable NSRangePointer */ IntPtr effectiveGlyphRange);
+
+		[Wrap ("GetLineFragmentRect (glyphIndex, IntPtr.Zero)")]
+		CGRect GetLineFragmentUsedRect (nuint glyphIndex);
+
+		[Sealed]
+		[Export ("lineFragmentUsedRectForGlyphAtIndex:effectiveRange:")]
+		CGRect GetLineFragmentUsedRect (nuint glyphIndex, out /* nullable NSRangePointer */ NSRange effectiveGlyphRange);
+
+		/* GetLineFragmentUsedRect (NSUInteger, NSRangePointer, bool) */
+		[iOS (9,0)]
+#if MONOMAC || XAMCORE_4_0
+		[Protected]
+#else
+		[Sealed][Internal]
+#endif
+		[Export ("lineFragmentUsedRectForGlyphAtIndex:effectiveRange:withoutAdditionalLayout:")]
+		CGRect GetLineFragmentUsedRect (nuint glyphIndex, /* nullable NSRangePointer */ IntPtr effectiveGlyphRange, bool withoutAdditionalLayout);
+
+		[iOS (9,0)]
+		[Wrap ("GetLineFragmentRect (glyphIndex, IntPtr.Zero)")]
+		CGRect GetLineFragmentUsedRect (nuint glyphIndex, bool withoutAdditionalLayout);
```

Comment:
should not this wrap `GetLineFragmentUsedRect` instead of `GetLineFragmentRect` ?